LogRRatioFormat.py

```diff
@@ -0,0 +1,68 @@
+from vcf.parser import _Format
+import numpy as np
+
+
+class LogRRatioFormat(object):
+    """
+    Generate log R ratio format information for VCF
+    """
+    def __init__(self, logger, log_r_ratio):
+        self._log_r_ratio = log_r_ratio
+        self._logger = logger
+
+    @staticmethod
+    def get_id():
+        return "LRR"
+
+    @staticmethod
+    def get_description():
+        return "Log R Ratio"
+
+    @staticmethod
+    def get_format_obj():
+        # arguments should be: ['id', 'num', 'type', 'desc']
+        return _Format(LogRRatioFormat.get_id(), 1, "Float", LogRRatioFormat.get_description())
+
+    def generate_sample_format_info(self, bpm_records, vcf_record, sample_name):
+        """
+        Get the sample log R ratio
+
+        Args:
+
+        Returns:
+            float: log R ratio
+        """
+        # if we have more than 1 BPMRecord, need to merge
+        log_r_ratio_list = []
+        if len(bpm_records) > 1:
```

Comment:
Is this conditional branch really needed? It seems like the code to handle the situation with a multiple BPM records could equivalently handle a single BPM record (possibly same question about determination of BAlleleFreq)

Reply:
No I don't think that it's needed, I'll remove it. Makes the code much cleaner.